packages/expo-updates/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- Logging and log reading. ([#18284](https://github.com/expo/expo/pull/18284) by [@douglowder](https://github.com/douglowder))
```

Comment:
Could you make it more descriptive? Also, it's not a new feature from the user perspective — it doesn't change the public API and doesn't bring any visible value for the user, so I would prefer moving this to "Others" category.
This reminded me that I should clarify it in our contributing guide about updating changelogs 😅

Reply:
True -- and the expotools add-changelog should support "Others" as well :)